### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards - ActivityUpdate]: Keyboard focus order of Date Picker in 'Set Due Date' under 'ActivityUpdate' is not logical using Arrow key.

### ISSUE THREAD

**Issue by Bhaskar301101** [microsoft/AdaptiveCards#8466]
### Target Platforms

Android

### SDK Version

2023.03.31.1

### Application Name

Adaptive Card

### Problem Description

Test Environment
Device: S21+ 5G
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.03.31.1
Tools: Logitech Bluetooth Keyboard

Repro Steps:

1. Launch the Adaptive cards app on Android.
2. Navigate to 'Import' control and select it.
3. Navigate to 'Sample cards' and select it.
4. Navigate to available dropdown under 'Sample cards' and select it.
5. Navigate all the controls under dropdown and select 'ActivityUpdate'.
6. Navigate to 'Visual' tab control and select it.
7. Navigate to 'Set Due Date' and activate it.
8. A Date Picker dialog will open.
9. Try navigating in the dates using arrow keys.
10. Observe the issue.

Actual Result:
Keyboard focus order of Date Picker in 'Set Due Date' under 'ActivityUpdate' is not logical. 
On pressing of the left/right arrow keys focus directly Moves to the next month date in that row.

Expected Result:
Keyboard focus order of Date Picker in 'Set Due Date' under 'ActivityUpdate' should be logical. On pressing of the left/right arrow keys focus should move to the next date in that same month. 
For example if focus is on 18 September on pressing of the right arrow key focus should move to 19 September and not to the next month.

User Experience:
Keyboard users would be confused if focus order of Date Picker in 'Set Due Date' under 'ActivityUpdate' is not logical.

WCAG Reference:
[[Focus Order - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/02.04.03)](https://www.w3.org/WAI/WCAG21/Understanding/focus-order)

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fforms.office.com%2fpages%2fresponsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-raians%40microsoft.com%7c6a2dded50c1c45e12a5608d9798bb984%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637674463301170115%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=3StFqNyebFAcprbeS8ca9BgKUtOHDRu8WJIr5iVU%2fXc%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/89897257/232708073-d02735ba-ec5a-4ccc-9b9a-dc8162ccd7fb.mp4

### Card JSON

```JSON
NA
```

**Comment by jwoo-msft** [comment]
it's also Android date picker issue, there is nothing we can do about it.
